src/pages/identity/Identity.js

```diff
@@ -1,10 +1,47 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import { connectIdmWallet } from 'react-idm-wallet';
+import styles from './Identity.css';
```

Comment:
Can we improve the order of the imports? e.g.: import of styles should be last.. also shared imports should be the first ones amongst the relative ones.